services/token-reward-service/.env

```
@@ -0,0 +1,16 @@
+NODE_ENV=development
+PORT=9009
+
+DB_USER=postgres
+DB_PASSWORD=password
+DB_HOST=localhost
+DB_PORT=5433
+DB_NAME=course_db
+
+WEB3_PROVIDER_URL=http://127.0.0.1:7545
+ACCOUNT_PRIVATE_KEY=0xaba9dede35ae3042e09949a82605a33f77b1602a6cc463ee7b829d663accf570
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ACCOUNT_PRIVATE_KEY key should go before the WEB3_PROVIDER_URL key

(UnorderedKey)